freqtrade/tests/conftest.py

```diff
@@ -12,7 +12,7 @@
 from telegram import Chat, Message, Update
 
 from freqtrade import constants
-from freqtrade.analyze import Analyze
+from freqtrade.analyze import parse_ticker_dataframe
```

Comment:
if analyze.py is removed, this will fail

Reply:
Fixed in 85fd4dd